lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -63,9 +73,48 @@ def _get_config(self, config_dict):
                                config_dict['ovs_internal_sampling_port_number'],
             internal_sampling_fwd_tbl=
                               config_dict['ovs_internal_sampling_fwd_tbl_number'],
+            multi_tunnel_flag=multi_tunnel_flag,
 
         )
 
+    def _get_gtp_port_no(self, port_name:str):
+        output = subprocess.check_output("sudo ovsdb-client dump Interface name ofport",
+                                          shell=True,).decode("utf-8")
```

Comment:
Lets use RYU lib to talk to OVSDB and list Interface table.

Reply:
Yes. We will do.

Reply:
Changes done. Could you please review it.